Jint/StringComparer.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Jint.Runtime.Interop
+{
+    public interface IStringComparer
```

Comment:
These interface and its implementation already exist in .NET, why are you reimplementing it?

Reply:
Ok, I will use the .NET implementation.